projects/examples/src/components/form-input/number-with-unit-form-input.example.component.scss

```diff
@@ -0,0 +1,7 @@
+.form-value-display {
```

Comment:
Why did we need these? Doesn't this mean the reusable component is not properly aligned? Shouldn't we fix it from the component itself?

Reply:
This is just cosmetic change for the example view - not to show the selected values too close under the inputs. It seemed to me better to have a little distance between them. 
There is no relation to the form itself.